jujugui/static/gui/src/app/components/status/status.js

```diff
@@ -5,23 +5,363 @@
 /** Status React component used to display Juju status. */
 class Status extends React.Component {
 
+  /**
+    Render the component.
```

Comment:
All other methods are called by render, so it seems to me good to have them right after render.
Also, why not putting all React specific methods on top and all user defined ones on the bottom?
Any specific reason to have render at the bottom?